README.md

```diff
@@ -193,13 +322,149 @@ generate RBIs for dynamic methods
+
+To generate the RBI file for the missing constants used in your application run the following command:
+
+```sh
+$ bin/tapioca todo
+
+Compiling sorbet/rbi/todo.rbi, this may take a few seconds... Done
+All unresolved constants have been written to sorbet/rbi/todo.rbi.
+Please review changes and commit them.
+```
+
+This will generate the file `sorbet/rbi/todo.rbi` defining all unresolved constants as empty modules. This file should be reviewed then committed in your repository.
 
-Tapioca supports loading command defaults from a configuration file. The default configuration
-file location is `sorbet/tapioca/config.yml` but this default can be changed using the `--config` flag
-and supplying an alternative configuration file path.
+<!-- START_HELP_COMMAND_TODO -->
+```shell
+$ tapioca help todo
 
-A configuration file must be a well-formed YAML file with top-level keys for the various Tapioca commands. Keys under each such top-level command should be the underscore version of a long option name for that command and the value for that key should be the value of the option.
+Usage:
+  tapioca todo
+
+Options:
+      [--todo-file=TODO_FILE]              # Path to the generated todo RBI file
+                                           # Default: sorbet/rbi/todo.rbi
+      [--file-header], [--no-file-header]  # Add a "This file is generated" header on top of each generated RBI file
+                                           # Default: true
+  -c, [--config=<config file path>]        # Path to the Tapioca configuration file
+                                           # Default: sorbet/tapioca/config.yml
+  -V, [--verbose], [--no-verbose]          # Verbose output for debugging purposes
+
+generate the list of unresolved constants
+```
+<!-- END_HELP_COMMAND_TODO -->
+
+#### Manually writing RBI definitions (shims)
+
+A _shim_ is a hand-crafted RBI file that tells Sorbet about constants, ancestors, methods, etc. that it can't understand statically and aren't already generated by Tapioca.
+
+These shims are usually placed in the `sorbet/rbi/shims` directory. From there, conventionally, you should follow the directory structure of the project to the file you'd like to shim. For example, say you had a `person.rb` file found at `app/models/person.rb`. If you were to add a shim for it, you'd want to create your RBI file at `sorbet/rbi/shims/app/models/person.rbi`.
+
+A shim might be as simple as the class definition with an empty method body as below:
+
+```ruby
+# typed: true
+
+class Person
+  sig { void }
+  def some_method_sorbet_cannot_find; end
+end
+```
+
+As you migrate to newer versions of Sorbet or Tapioca, some shims may become useless as Sorbet's internal definitions for Ruby's core and standard library is enhanced or Tapioca is able to generate definitions for new DSLs. To avoid keeping outdated or useless definitions inside your application shims, Tapioca provide the `check-shims` command:
```

Comment:
```suggestion
As you migrate to newer versions of Sorbet or Tapioca, some shims may become useless as Sorbet's internal definitions for Ruby's core and standard library is enhanced or Tapioca is able to generate definitions for new DSLs. To avoid keeping outdated or useless definitions inside your application shims, Tapioca provides the `check-shims` command:
```